redhat-appstudio/application-service
# Update HAS for KCP

- Makes a number of changes to the kustomization manifests for deploying HAS on KCP
   - Separates the APIExport, APIResourceSchema, and APIBindings from the rest of the KCP manifests, to simplify deployment via infra-deployments
   - Adds a new `deploy-kcp` makefile target for deploying HAS on KCP
   - Updates the `docs/kcp.md` doc on deploying HAS on KCP
- Updates the KCP libraries in HAS for compatibility on CPS

### How to Test

1) Build the changes in this PR and push up to Quay
1) Create an OpenShift cluster (the workload cluster)
2) Follow the CPS onboarding document that outlines logging in kcp-stable, and setting up a synctarget / workload cluster
3) Follow the instructions in `docs/kcp.md`, and make sure ‘IMG` is set in your terminal when running `make deploy-kcp`
